docs/docs/advanced-topics/2-transactions.md

```diff
@@ -74,6 +74,18 @@ await queryRunner.connect()
 // now we can execute any queries on a query runner, for example:
 await queryRunner.query("SELECT * FROM users")
 
+// **NOTE:** if you need to access multiple recordsets from a single query, pass true as the third parameter to the `query` method:
```

Comment:
The topic of this doc page is Transactions. This is not the right place to document the structured result feature.

There is a [Query Runner](https://typeorm.io/docs/query-runner) page where it could go instead.